olive/passes/onnx/onnxscript_dla_transforms.py

```diff
@@ -0,0 +1,111 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+import logging
+from pathlib import Path
+from typing import Dict, Type
+
+import onnx
+import onnxscript
+from onnxscript.rewriter import pattern
+
+from olive.hardware.accelerator import AcceleratorSpec
+from olive.model import ONNXModelHandler
+from olive.model.utils import resolve_onnx_path
+from olive.passes import Pass
+from olive.passes.onnx.common import get_external_data_config, model_proto_to_olive_model
+from olive.passes.pass_config import BasePassConfig, PassConfigParam
+
+logger = logging.getLogger(__name__)
+
+
+class MatMulToConvTransform(Pass):
```

Comment:
Can you add a docstring here to explain what does this pass do? That will be useful for auto-generated doc.

Reply:
Sure, description has been added